lib/vertica/query.rb

```diff
@@ -18,7 +18,8 @@ class Vertica::Query
   #   to receive data for a `COPY ... FROM STDIN` statement.
   def initialize(connection, sql, row_handler: nil, copy_handler: nil)
     @connection, @sql = connection, sql
-    @buffer = row_handler.nil? && copy_handler.nil? ? [] : nil
+    @enable_buffering = row_handler.nil? && copy_handler.nil?
```

Comment:
I notice we have `buffer_rows?`already. Can we make use of that, or change the implementation to  simply return `@enable_buffering`?